build-support/scripts/devtools.sh

```diff
@@ -51,6 +51,10 @@ function main {
                 codegen_install
                 return 0
                 ;;
+            -copyright )
+                copyright_install
+                return 0
+                ;;
```

Comment:
Instead of having a new `--copywrite` category, can we get by with adding `copywrite` to the existing `-lint`? It looks like `deepcopy` is already under `-codegen` which makes sense.

Reply:
I'm not sure lint is the right category here, as lint tools are not supposed to change file content. I thought that the closest category is code-gen as we are generating file headers.